### PR TITLE
Harden pikepdf version checking

### PR DESCRIPTION
**Pull request by m-holger**
Avoid future problems with eg 10.0.0 testing as less than 8.0.0.

### CODE REVIEW

File: pdfarranger/exporter.py [hunk 1]
```diff
@@ -447,7 +449,7 @@ def generate_booklet(pdfqueue, tmp_dir, pages):
             )
 
         # workaround for pikepdf <= 2.6.0. See https://github.com/pikepdf/pikepdf/issues/174
-        if pikepdf.__version__ < '2.7.0':
+        if pre_pike_2_7:
```
Comment (dreua) [inline pdfarranger/exporter.py:452]: Good catch:
```
>>> '2.10.0' < '2.8.0'
True
```

File: pdfarranger/exporter.py [hunk 2]
```diff
@@ -399,6 +400,7 @@ def num_pages(filepath):
 
 
 def generate_booklet(pdfqueue, tmp_dir, pages):
+    pre_pike_2_7 = version.parse(pikepdf.__version__) < version.Version('2.7.0')
```
Comment (dreua) [inline pdfarranger/exporter.py:403]: Why not use parse or Version constructor on both sides? What is the difference actually?

File: setup.py [hunk 3]
```diff
@@ -117,7 +117,7 @@ def run(self):
     entry_points={
         'console_scripts': ['pdfarranger=pdfarranger.pdfarranger:main']
     },
-    install_requires=['pikepdf>=1.17.0','python-dateutil>=2.4.0'],
+    install_requires=['pikepdf>=1.17.0','python-dateutil>=2.4.0', 'packaging'],
```
Comment (dreua) [inline setup.py:120]: packaging is also required by pikepdf :heavy_check_mark:

File: pdfarranger/exporter.py (hunk 2 above)
Comment (m-holger) [reply pdfarranger/exporter.py:403]: I believe there is no real difference. Using both is a stylistic preference - it feels more natural to me for a literal and parse for an unknown version, but if you want me to use one or the other I don't have a problem with changing it.

**Pull request merged**